src/components/Table/Budget/BudgetLineItemTable/BudgetLineItemTable.tsx

```diff
@@ -73,15 +73,21 @@ const getInitialValues = (
 	budgetLineItem: IBUDGET_LINE_ITEM_RESPONSE | null
 ): IBudgetTrackingLineitem => {
 	return {
-		amount: budgetLineItem ? budgetLineItem.amount : 0,
-		note: budgetLineItem ? budgetLineItem.note : "",
-		budget_targets_project: budgetLineItem ? budgetLineItem.budget_targets_project.id : "",
-		annual_year: budgetLineItem ? budgetLineItem.annual_year.id : "",
-		reporting_date: getTodaysDate(budgetLineItem ? budgetLineItem.reporting_date : undefined),
-		id: budgetLineItem ? budgetLineItem.id : "",
-		grant_periods_project: budgetLineItem ? budgetLineItem.grant_periods_project.id : "",
-		fy_org: budgetLineItem ? budgetLineItem?.fy_org?.id : "",
-		fy_donor: budgetLineItem ? budgetLineItem?.fy_donor?.id : "",
+		amount: budgetLineItem?.amount ? budgetLineItem?.amount : 0,
+		note: budgetLineItem?.note ? budgetLineItem?.note : "",
+		budget_targets_project: budgetLineItem?.budget_targets_project.id
+			? budgetLineItem?.budget_targets_project.id
+			: "",
+		annual_year: budgetLineItem?.annual_year?.id ? budgetLineItem?.annual_year?.id : "",
+		reporting_date: getTodaysDate(
+			budgetLineItem?.reporting_date ? budgetLineItem?.reporting_date : undefined
```

Comment:
`budgetLineItem?.reporting_date ? budgetLineItem?.reporting_date : undefined` expression like these can be simplified  to ` budgetLineItem?.reporting_date  || undefined`. 
The later one is more more readable.